Classes/MSAISender.m

```diff
@@ -49,37 +49,29 @@ - (void)registerObservers{
                   usingBlock:^(NSNotification *notification) {
                     typeof(self) strongSelf = weakSelf;
                     dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Test performance with different priorities for the queue as high/low prio may have a positive impact